packages/ui/components/ui/identity-key-component.tsx

```diff
@@ -14,8 +17,10 @@ import { bech32IdentityKey } from '@penumbra-zone/bech32/src/identity-key';
 export const IdentityKeyComponent = ({ identityKey }: { identityKey: IdentityKey }) => {
   return (
     <div className='flex min-w-0 items-center gap-2'>
-      <div className='min-w-0 truncate font-mono text-muted-foreground'>
-        {bech32IdentityKey(identityKey)}
+      <div className='min-w-0 truncate font-mono'>
+        <span className='text-muted-foreground'>{PREFIX_AND_SEPARATOR}</span>
+
+        {bech32IdentityKey(identityKey).replace(PREFIX_AND_SEPARATOR, '')}
```

Comment:
Now, the ID part of the identity key is highlighted:
<img width="579" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/b255eb00-a7ca-44c2-b538-7b852f532b58">